elastic-search-support/src/main/resources/globalSearchCasesMapping.json

```diff
@@ -232,264 +232,6 @@
         }
       }
     },
-    "data_classification": {
-      "type": "object",
```

Comment:
Though we don't use this in the response but  i just wonder access control code in ElasticSearchOperation code required `data_classification` ?

Reply:
It should still be in the response from the ES search, just not available to serach upon.

Reply:
Ok, does ElasticSearch will auto create / update required default analysers based on first record on the fly?  What if we got second case record where structure has new fields (searchParty etc) ?

Reply:
After thinking a while you're @mattnayler we probably don't need any predefined mappings for `data_classification` as they are all String properties and also we never search upon them.

Reply:
As discussed I have re-added `data_classification` but set it as `"enabled": false` to stop it being added by dynamic mapping, see  in b1158d458b13f673282ed1c8dc5ec79905e3470e.